src/plugins/DifficultyPlugin.js

```diff
@@ -18,35 +14,115 @@ export class DifficultyPlugin extends BasePlugin {
+      timersScale: defaultTimersScale,
+      inputCount: defaultInputCount,
+    };
 
-    this.difficultySlider = new Slider({
-      slider: difficultySlider,
-      control: DifficultyPlugin.difficultyKey,
-      min: DIFFICULTY_SLIDER_MIN,
-      max: DIFFICULTY_SLIDER_MAX,
-      step: DIFFICULTY_SLIDER_STEP,
-      defaultValue: this.difficulty
-    });
+    this.sliders = {
+      //Hit Area Scale
+      hitAreaScaleSlider: new Slider({
+        slider: hitAreaScaleSlider,
+        control: DifficultyPlugin.hitAreaScaleKey,
+        defaultValue: this.values.hitAreaScale
+      }),
 
-    this.difficultySlider.enableSliderEvents(
-      this.onDifficultyChange.bind(this)
-    );
+      //Drag Threshold Scale
+      dragThresholdScaleSlider: new Slider({
+        slider: dragThresholdScaleSlider,
+        control: DifficultyPlugin.dragThresholdScaleKey,
+        defaultValue: this.values.dragThresholdScale
+      }),
+
+      //Health
+      healthSlider: new Slider({
+        slider: healthSlider,
+        control: DifficultyPlugin.healthKey,
+        defaultValue: this.values.health
+      }),
+
+      //Object Count
+      objectCountSlider: new Slider({
+        slider: objectCountSlider,
+        control: DifficultyPlugin.objectCountKey,
+        defaultValue: this.values.objectCount
+      }),
+
+      //Completion Percentage
+      completionPercentageSlider: new Slider({
+        slider: completionPercentageSlider,
+        control: DifficultyPlugin.completionPercentageKey,
+        defaultValue: this.values.completionPercentage
+      }),
+
+      //Speed Scale
+      speedScaleSlider: new Slider({
+        slider: speedScaleSlider,
+        control: DifficultyPlugin.speedScaleKey,
+        defaultValue: this.values.speedScale
+      }),
+
+      //Timer Scale
+      timersScaleSlider: new Slider({
+        slider: timersScaleSlider,
+        control: DifficultyPlugin.timersScaleKey,
+        defaultValue: this.values.timersScale
+      }),
+
+      //Input Count
+      inputCountSlider: new Slider({
+        slider: inputCountSlider,
+        control: DifficultyPlugin.inputCountKey,
+        defaultValue: this.values.inputCount
+      }),
+
+    };
+
+    Object.keys(this.sliders).forEach(key => {
+      this.sliders[key].enableSliderEvents(() => {
+        this.onDifficultyChange(this.sliders[key].control);
+      });
+      this.values[this.sliders[key].control] = this.sliders[key].value;
+    });
   }
 
   /**
    * @memberof ControlsPlugin
    */
-  onDifficultyChange() {
-    this.difficulty = this.difficultySlider.sliderRange(
-      Number(this.difficultySlider.slider.value)
+  onDifficultyChange(control) {
+    this.values[control] = this.sliders[`${control}Slider`].sliderRange(
+      Number(this.sliders[`${control}Slider`].slider.value)
     );
-    this.sendProperty(DifficultyPlugin.difficultyKey, this.difficulty);
+    this.sendProperty(DifficultyPlugin[`${control}Key`], this.values[control]);
```

Comment:
We definitely can, I used the keys to keep it consistent with the older plugins as they all use the keys. But since these are all using control as an option anyway there's no need.